app/src/App.tsx

```diff
@@ -53,68 +51,83 @@ export default () => {
 						gridTemplateRows: '60px calc(50% - 60px) 60px calc(50% - 60px)'
 					}}
 				>
-					<div style={{ border: '1px solid blue', gridArea: '1 / 1 / span 1 / span 3' }}>
+					<NeonBox style={{ gridArea: '1 / 1 / span 1 / span 3' }}>
 						<Title />
-					</div>
+					</NeonBox>
 					<StatsWrapper>
 						<div style={{ height: '100%', minHeight: '300px', position: 'relative' }}>
 							<Stats />
 							<Saves />
 						</div>
 					</StatsWrapper>
-					<div style={{ border: '1px solid purple', gridArea: '2 / 2 / span 1 / span 1' }}>
+					<NeonBox style={{ gridArea: '2 / 2 / span 1 / span 1' }}>
 						<Actions />
-					</div>
-					<div style={{ border: '1px solid red', gridArea: '3 / 1 / span 1 / span 2' }}>
+					</NeonBox>
+					<NeonBox style={{ gridArea: '3 / 1 / span 1 / span 2' }}>
 						<HealthBar />
-					</div>
-					<InventoryWrapper>
-						<Inventory />
-					</InventoryWrapper>
-					<CyberneticWrapper>
-						<Cybernetics />
-					</CyberneticWrapper>
-					<SkillsWrapper style={{ border: '1px solid blue', gridArea: '2 / 3 / span 3 / span 1' }}>
-						<Skills />
-					</SkillsWrapper>
+					</NeonBox>
+					<InventoryWrapper component={Inventory} />
+					<CyberneticWrapper component={Cybernetics} />
+					<SkillsWrapper component={Skills} />
 				</div>
 			</characterContext.Provider>
 		</>
 	);
 };
 
-const StatsWrapper = styled.div`
+const NeonBoxStyle = css`
```

Comment:
Can I ask why you made this a const, and didn't just directly add the styling to the below styled div? I don't see any benefit, and I think it hurts readability.